app/models/manageiq/providers/ansible_tower/embedded_ansible_worker.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::AnsibleTower::EmbeddedAnsibleWorker < MiqWorker
```

Comment:
@Fryguy I was thinking it would be more of a separate thing because it's related to running the service on the appliance rather than with talking to the provider's API.